operations/service_web.go

```diff
@@ -55,7 +55,7 @@ func startWebService() cli.Command {
 			// Create the user manager before setting up job queues to allow
 			// background reauthorization jobs to start.
 			if err = setupUserManager(env, settings); err != nil {
-				return errors.Wrap(err, "could not set up user manager")
+				grip.EmergencyFatal(errors.Wrap(err, "could not set up user manager"))
```

Comment:
This exits, which also would make the app fail to start.

Reply:
Also, if you do this, it won't solve the undelrying problem because the result will be that there will be no auth whatsoever (including Okta), and neither API users nor regular users will be able to auth.